dias/analyzers/source_spectra_analyzer.py

```diff
@@ -0,0 +1,733 @@
+"""Extracts flux of a given source, as a function of freq, when the source is right at the zenith.
+
+.. currentmodule:: dias.analyzers.source_spectra_analyzer
+
+Classes
+=======
+
+.. autosummary::
+    :toctree: generated/
+
+    SourceSpectraAnalyzer
+
+Functions
+=========
+
+.. autosummary::
+    :toctree: generated/
+
+    fit_point_source_transit
+
+"""
+
+import os
+import gc
+import time
+import subprocess
+from datetime import datetime
+from collections import Counter, defaultdict
+
+import h5py
+import numpy as np
+import scipy.constants
+from scipy.optimize import curve_fit
+
+from caput import config
+from chimedb import data_index
+from ch_util import ephemeris, andata, tools, cal_utils, fluxcat
+from dias import CHIMEAnalyzer, exception
+from dias import __version__ as dias_version_tag
+from dias.utils.string_converter import str2timedelta
+from dias.utils.helpers import get_cyl
+
+########################################################
+# main analyzer task
+########################################################
+
+
+class SourceSpectraAnalyzer(CHIMEAnalyzer):
+    """Extracts flux of a given source transit, as a function of freq, when the source is right at the zenith.
+
+    If the flux of the source is similar to what we expect from other measurements, our calibration is doing a good job.
+
+    Metrics
+    -------
+    dias_task_<task_name>_source_analyzer_run_total
```

Comment:
No need, dias also provides this in `dias/task.py`. Maybe we should mention that in the documentation....